index.html

```diff
@@ -13,6 +13,11 @@
 		<link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
 		<meta name="theme-color" content="#000000" />
 		<title>Smart Shopping List</title>
+		<!-- Script for fontawesome icons -->
+		<script
+			src="https://kit.fontawesome.com/c9c5205463.js"
+			crossorigin="anonymous"
+		></script>
```

Comment:
Just wondering what this is for?

Reply:
@sarahmjenkins fontawesome is a site full of icons we can use. It's great because the icons are actually fonts themselves. I used it for the arrow buttons.

Reply:
Same question.  Thanks in advance for letting us know